packages/client/src/lib.js

```diff
@@ -215,6 +217,50 @@ class Web3Storage {
     return res.json()
   }
 
+  /**
+   * @param {Service} service
+   * @param {{before: string, size: number}} opts
```

Comment:
We should really have a type for this option so we can explain that `before` is a date as an ISO string and `size` is the max page size, not the max results to retrieve.